src/SurveyService.Business/Commands/Question/GetQuestionCommand.cs

```diff
@@ -0,0 +1,106 @@
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.SurveyService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.SurveyService.Business.Commands.Question.interfaces;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Mappers.Responses.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question.Filters;
+using LT.DigitalOffice.SurveyService.Models.Dto.Responses.Question;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Business.Commands.Question;
+
+public class GetQuestionCommand : IGetQuestionCommand
+{
+  private readonly IQuestionRepository _repository;
+  private readonly IQuestionResponseMapper _questionResponseMapper;
+  private readonly IResponseCreator _responseCreator;
+  private readonly IUserService _userService;
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public GetQuestionCommand(
+    IHttpContextAccessor httpContextAccessor,
+    IQuestionRepository questionRepository, 
+    IQuestionResponseMapper questionResponseMapper,
+    IResponseCreator responseCreator,
+    IUserService userService)
+  {
+    _httpContextAccessor = httpContextAccessor;
+    _repository = questionRepository;
+    _questionResponseMapper = questionResponseMapper;
+    _responseCreator = responseCreator;
+    _userService = userService;
+  }
+  
+  public async Task<OperationResultResponse<QuestionResponse>> ExecuteAsync(GetQuestionFilter filter)
+  {
+    OperationResultResponse<QuestionResponse> response = new();
+    if (filter is null)
+    {
+      return _responseCreator.CreateFailureResponse<QuestionResponse>(
+        HttpStatusCode.BadRequest,
+        new List<string> { "You must enter 'questionid'." });
+    }
+
+    DbQuestion dbQuestion = await _repository.GetAsync(filter);
```

Comment:
add check: if dbQuestion is null, return NotFound

Reply:
Done